src/widgets/settingspages/GeneralPage.cpp

```diff
@@ -314,7 +316,7 @@ void GeneralPage::initLayout(GeneralPageView &layout)
     layout.addCheckbox("Enable emote auto-completion by typing :",
                        s.emoteCompletionWithColon);
     layout.addDropdown<float>(
-        "Size", {"0.5x", "0.75x", "Default", "1.25x", "1.5x", "2x"},
+        "Size", {"0.5x", "0.75x", "Default", "1.25x", "1.5x", "2x", "3x", "4x"},
```

Comment:
Do you know why this was added/necessary for this PR?

Reply:
I suppose since we have emotes with larger sizes (#3812) we can display them bigger.

For reference:
`emoteScale` is used in
* https://github.com/Chatterino/chatterino2/blob/c692dd9b4461f486762f44cc007125c229d28592/src/messages/ImageSet.cpp#L64
* https://github.com/Chatterino/chatterino2/blob/c692dd9b4461f486762f44cc007125c229d28592/src/messages/MessageElement.cpp#L191
* https://github.com/Chatterino/chatterino2/blob/c692dd9b4461f486762f44cc007125c229d28592/src/messages/MessageElement.cpp#L493